contracts/ClearingHouse.sol

```diff
@@ -863,15 +863,18 @@ contract ClearingHouse is
             );
         }
 
-        // check margin ratio after swap: mmRatio for closing position; else, imRatio
+        // check margin ratio after swap: mmRatio for closing position
```

Comment:
Not sure if we can combine these two conditions and removed `CH_NEMRM` to make reducing and clsosing position all covered by  `CH_NEFCM` to prevent 14204 related issue.

It seems that `CH_NEFCM` can cover `CH_NEMRM`'s scenario.